frontend/src/tests/application-tests.js

```diff
@@ -6,20 +6,72 @@ import {
     postingPropTypes,
 } from "./utils";
 import { databaseSeeder } from "./setup";
+import axios from "axios";
+import fs from "fs";
+import path from "path";
+import md5 from "md5";
 
 export function applicationsTests({ apiGET, apiPOST }) {
     let session;
     let position;
     let posting = {};
     let adminUser;
-    let surveyData;
+
+    const BACKEND_BASE_URL = "http://backend:3000";
+
+    // Returns hashes rather than expecting their equality as possible fail is untracable back to the test case
+    async function getMD5Hashes(url_token, file_type) {
```

Comment:
Do we want a better dosctring for these? I am not sure what `file_type` I can provide based on the current comment. The `url_token` here is the token to get on the `/public/files/` path, right?